webroot/src/components/Page/PageMainNav/PageMainNav.ts

```diff
@@ -45,21 +46,25 @@ class PageMainNav extends Vue {
         return this.userStore.currentCompact;
     }
 
+    get isLoggedInAsStaff(): boolean {
+        return this.$store.getters['user/authType']() === AuthTypes.STAFF;
```

Comment:
Based on how this is used in the links below and how it's named, Would it make more sense to just also include checking `this.isLoggedIn` here?

Something like:

```
return this.isLoggedIn && this.$store.getters['user/authType']() === AuthTypes.STAFF;
```

That way using `isLoggedInAsStaff` elsewhere also implies the `isLoggedIn...` part?